peps/pep-0791.rst

```diff
@@ -107,60 +163,25 @@ Reference Implementation
 `python/cpython#133909 <https://github.com/python/cpython/pull/133909>`_
 
 
-Open Issues
-===========
+Rejected ideas
+==============
 
 Module name
 -----------
 
-The chosen name seems consistent with one existing domain-specific mathematical module:
-:external+py3.14:mod:`cmath` (for complex numbers).
-
-We note the `Imath
-<https://github.com/AcademySoftwareFoundation/Imath>`_ C++ library includes
-Python bindings with the same name.  There is also an :pypi:`imath` project on
-PyPI, but only with two releases, with the most recent one four years ago.  Its
-repository is no longer accessible.
-
-`Polling showed <https://discuss.python.org/t/91337/35>`_ ``intmath`` as another
-popular name.  The argument made was that the normal mathematical spelling of
-the imaginary unit is ``i``, which makes ``imath`` ambiguous.  It also has no conflict
-with any PyPI module.  On the other hand, ``intmath`` may be confused with
-interval math or numerical integration.
+`Polling showed <https://discuss.python.org/t/92548/67>`_ ``intmath`` as most
+popular candidate with ``imath`` as a second winner.
 
 Other proposed names include ``ntheory`` (like SymPy's submodule),
-``integermath`` and ``imaths``.
-
+``integermath``, ``zmath``, ``dmath`` and ``imaths``.
 
-Module scope and possible extensions
-------------------------------------
-
-Unless we can just provide bindings to some well supported mathematical library
-like the GMP, the module scope should be limited.  For example, no primality
-testing and factorization, as production-quality implementatons will require a
-decent mathematical background from contributors and belongs rather to
-specialized libraries.
-
-Some possible additions, among those proposed in the initial discussion thread
-(see also issue
-`python/cpython#81313 <https://github.com/python/cpython/issues/81313>`_):
-
-* ``ceil_div()`` --- for integer ceiling divide, see
-  `relevant discussion thread <https://discuss.python.org/t/91269>`_.
-* ``gcdext()`` --- to solve linear `Diophantine equation <https://en.wikipedia.org/wiki/Diophantine_equation>`_ in two variables (the
-  :external+py3.14:class:`int` implementation actually includes an extended
-  Euclidean algorithm)
-* ``isqrt_rem()`` --- to return both an integer square root and a remainder (which is non-zero only if
-  the integer isn't a perfect square)
-* ``ilog()`` --- integer logarithm, :external+py3.14:func:`math.log`
-  has special handling for integer arguments.  It's unique (with respect to other module
-  functions) and not documented so far, see issue
-  `python/cpython#120950 <https://github.com/python/cpython/issues/120950>`_.
-* ``fibonacci()`` --- `Fibonacci sequence <https://en.wikipedia.org/wiki/Fibonacci_sequence>`_.
+As a variant, the new module can be added as a submodule of the
+:external+py3.14:mod:`math`: ``integer`` (most preferred), ``discrete``
+or ``ntheory``.
 
 
-Rejected ideas
-==============
+:external+py3.14:func:`~math.isqrt` renaming
```

Comment:
I don't think that it's a good idea to add links in titles. I suggest to just write "isqrt renaming" without any formatting.